epytope/EpitopePrediction/External.py

```diff
@@ -168,14 +170,15 @@ def predict(self, peptides, alleles=None, command=None, options=None, **kwargs):
                         raise RuntimeError(e)
 
                     res_tmp = self.parse_external_result(tmp_out_path)
-                    for allele, scores in res_tmp.items():
+                    for allele_string, scores in res_tmp.items():
+                        allele = alleles_string[allele_string]
                         if allele not in result.keys():
                             result[allele] = {}
                         for scoretype, pep_scores in scores.items():
                             if scoretype not in result[allele].keys():
                                 result[allele][scoretype] = {}
                             for pep, score in pep_scores.items():
-                                result[allele][scoretype][pep] = score
+                                result[allele][scoretype][pep_seqs[pep]] = score
```

Comment:
Why did this change? Seems unrelated...

Reply:
I observed the same issue with newly created `Allele Obj` also with `Peptide Obj`. Therefore I used this `pep_seqs` dictionary to save the information of the `Peptide Obj(s)`